### DISCUSSION TITLE
Do we want to include more of a discussion of ML applications of DeepBench in the JOSS paper?

### DISCUSSION THREAD

**Discussion by beckynevin** [deepskies/DeepBench#72] [General]
@bnord I know the JOSS paper is supposed to be software focused. I was wondering if it would be possible / a good idea to slip in some mention of the fact that this package can be used for more deterministic traditional ML classifiers as well as generative noise aware tools as well as SBI type tools (because it's a simulation). I feel like this is a real strength of having the noise model built in.

**Comment by voetberg** [comment]
I agree! The original ICML paper discussed a little about how it can be used as a diagnosis/benchmark, but I hadn't really thought a lot about SBI at that point, but it seems like a logical continuation of how much detail we pay the parameter control.

**Reply by bnord** [reply]
> can be used for more deterministic traditional ML classifiers as well as generative noise aware tools as well as SBI type tools (because it's a simulation). I feel like this is a real strength of having the noise model built in.

I think that's a good idea. I believe it would go into the statement of need section? Do you want to write a few sentences? I can then go mess with it a bit?

**Comment by beckynevin** [comment]
Awesome, I added a few sentences to the joss-paper branch :)
